---
layout: page
title: "WOLF"
description: "ERC Project"
header-img: "img/wolf-logo.jpg"
---


Presentation
-------------------
WOLF (Wasserstein Flow Learning for Multi-Omics) is a research project financed by the [ERC](http://erc.europa.eu/) [Advanced Grant](https://erc.europa.eu/apply-grant/advanced-grant) program. 

Single cell molecular profiling allows to map cellular development at an unprecedented level of detail. Optimal transport (OT) enables the analysis of this dynamical process as a trajectory inference problem, using OT flows. These flows treat cells as particles evolving on an energy landscape over an "omics" space (such as transcriptomic, epigenomic, proteomic and location). Learning this model from large scale omics datasets poses however formidable mathematical and computational challenges, which will be tackled by WOLF. The first one is the joint learning of both the gene embedding space and the energy landscape. Existing approaches use ad-hoc Euclidean embeddings, ignoring biological relationships between genes. WOLF will develop a new type of non-Euclidean OT flows, which takes into account complex genetic relations. The second challenge is the fusion of multiple omics dataset (for instance transcriptomics, proteomic and space) without having access to an explicit pairing between the cells across the omics. Multi-omics is the next frontier in developmental analysis, and the corresponding trajectories cannot be captured with existing OT flows. WOLF will develop a new class of multi-linear OT flows where interaction terms couple particles together across different omics. These advances will be integrated in an efficient computational package where the parameters of the models are learned using parallelizable OT flow solvers. Leveraging the connexion between OT flows and attention mechanisms in deep learning, these methods will be approximated using transformers architectures and optimized using implicit differentiation. These theoretical and numerical contributions will work hand in hand to offer the first comprehensive framework for multi-omics trajectory inference. This will unlock biological findings for the characterization of developmental molecular pathways and the understanding of disease mechanisms.

Collaborators
-------------------

* [Laura Cantini](https://research.pasteur.fr/fr/member/laura-cantini/): CNRS and Pasteur.
* [Remi Gribonval](https://people.irisa.fr/Remi.Gribonval/): INRIA.
* [Jérome Malick](https://ljk.imag.fr/membres/Jerome.Malick/): CNRS, Grenoble.
* [François-Xavier Vialard](https://www.ceremade.dauphine.fr/~vialard/): Univ. Paris-Est.
* [Clarice Poon](https://cmhsp2.github.io/): Bath University.
* [Pierre Ablin](https://pierreablin.com/): Apple.
* [Maarten V. de Hoop](https://maartendehoop.rice.edu/): Rice University.

Visitors
-------------------

* [Demba Ba](https://seas.harvard.edu/person/demba-ba): Harvard
* [Hugo Lavenant](https://hugolav.github.io/): Boconi

PhDs and post-docs
-------------------

* [Anna Audit](https://research.pasteur.fr/fr/member/anna-audit/): PhD
* [Raphaël Barboni](https://scholar.google.com/citations?user=tEu2ONgAAAAJ&hl=en): PhD.
* [Valérie Castin](https://samuelhurault.netlify.app/), PhD.
* [Samuel Hurault](https://samuelhurault.netlify.app/): postdoc.
* [Jérémie Kalfon](https://www.jkobject.com/): PhD.
* [Francisco Lima](https://www.it.pt/Members/Index/35492), Postdoc.
* [Sibylle Marcotte](https://www.math.ens.psl.eu/instructor/sibylle-marcotte/), PhD.
* [Romain Petit](https://rpetit.github.io/): Postdoc.
* [Jules Samaran](https://github.com/jules-samaran), PhD.

Former students
-------------------


Events
-------------------

* [Towards a theory for typical-case algorithmic hardness](https://leshouches-algorithms.github.io/), Les Houches, Jan 2025.